### ISSUE TITLE
untrusted option is confusing

### ISSUE THREAD

**Issue by directionless** [mtrojnar/osslsigncode#67]
Thanks so much for picking up the mostly abandoned `osslsigncode`. I noticed a new option -- `untrusted`. It sounds like it's about something that's untrusted, but the underlying source code looks like it's a CA path for timestamp servers? I'm confused about what it's supposed to be.

**Comment by mtrojnar** [comment]
@olszomal Can you please handle this?

**Comment by olszomal** [comment]
Option `-untrusted` specifies a file of Time-Stamp Authority CA certificates to use if you don't have this certificates in the default  certificates store.

This option is  inspired by:
[https://www.openssl.org/docs/man1.1.1/man1/openssl-ts.html](https://www.openssl.org/docs/man1.1.1/man1/openssl-ts.html)
`-untrusted cert_file.pem`
_Set of additional untrusted certificates in PEM format which may be needed when building the certificate chain for the TSA's signing certificate. This file must contain the TSA signing certificate and all intermediate CA certificates unless the response includes them_.

This option name may seems confusing. Any suggestions would be appreciated.

**Comment by directionless** [comment]
I'm not very clever at naming, I'd probably call it `additional_tsa_cafile` or `CAfile_tsa` or similar. If it's easier, I'm happy to PR it, it looks self contained.

**Comment by hippie68** [comment]
I compiled the new osslsigncode 2.1.0 on Debian 10. While the old version 1.7.1 that ships with Debian 10 does not require CA files to be entered by hand, the new one does. It seems it does not search for the files nor does it look after /etc/ssl/certs/ca-certificates.crt.

Is this a new behavior or were paths hard-compiled into Debian's 1.7.1?

So I had to guess what files I might have to enter for file verification. I did this:

    osslsigncode verify -CAfile /etc/ssl/certs/ca-certificates.crt -untrusted /etc/ssl/certs/ca-certificates.crt file.exe

... and then file.exe passed all test.

Is /etc/ssl/certs/ca-certificates.crt a two-in-one file that servers both CA and TSA?
